.github/workflows/clair-in-ci.db.yaml

```diff
@@ -0,0 +1,79 @@
+name: Clair in CI DB - test
+
+# This workflow tests latest database version of clair-in-ci and
+# pushes it to app-studio registry in case all tests passed
+
+on:
+  repository_dispatch:
+    types: trigger_workflow
+    paths:
+      - clair-in-ci/**
+      - .github/workflows/clair-in-ci-db.yaml
+
+env:
+  REGISTRY: quay.io/redhat-appstudio
+  IMAGE_NAME: clair-test-build
+  LATEST_TAG: latest
+  PREVIOUS_TAG: previous
+  TO_TEST: ${{ github.event.client_payload.tag }}
```

Comment:
Please name it `image_tag` so it's clear it's not a git tag. I was thinking for a while, why git tag :D